ui/components/ui/loading-heartbeat/index.js

```diff
@@ -24,6 +26,11 @@ export default function LoadingHeartBeat() {
         e.preventDefault();
         e.stopPropagation();
       }}
+      style={{ backgroundColor }}
```

Comment:
Is there a reason why we have to fill in this div with any kind of color when the item in question is being loaded? I noticed that in the styles for LoadingHeartbeat we set a default background color of `#fff`. Do we need this at all? Can we just fade out whatever text is being loaded and fade it back in?

Reply:
Hey @mcmire : the way this loader currently works is it is absolutely positioned on top of relative container and covers it. It changes its opacity (animates) thus the text or anything in container blinks.

Background colour of loader needs to be same as container this I made this change, this fixes point-8 here: https://github.com/MetaMask/metamask-extension/issues/13346

Reply:
Oh right. I still don't like that, but I guess maybe there's a reason for it. Okay 👍🏻

Reply:
We can refactor to improve this loader, but yep currently that is out of scope.